seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/sink/writer/OrcWriteStrategy.java

```diff
@@ -74,13 +74,14 @@ public void write(@NonNull SeaTunnelRow seaTunnelRow) {
         Writer writer = getOrCreateWriter(filePath);
         TypeDescription schema = buildSchemaWithRowType();
         VectorizedRowBatch rowBatch = schema.createRowBatch();
-        int i = 0;
         int row = rowBatch.size++;
-        for (Integer index : sinkColumnsIndexInRow) {
-            Object value = seaTunnelRow.getField(index);
-            ColumnVector vector = rowBatch.cols[i];
-            setColumn(value, vector, row);
-            i++;
+        for (int i = 0; i < sinkColumnsIndexInRow.size(); i++) {
+            Integer index = sinkColumnsIndexInRow.indexOf(i);
+            if (index != null) {
+                Object value = seaTunnelRow.getField(index);
+                ColumnVector vector = rowBatch.cols[i];
+                setColumn(value, vector, row);
+            }
```

Comment:
Could you please explain what't the meaning of this? I have some concern. The sinkColumnsIndexInRow array holds the field index values to be sent to downstream fields in the upstream row, why need invoke the `indexOf` method for it?

Reply:
It's my mistake to use method indexOf() instead of get(). 
Already updated. 
Thks a lot.